website/docs/reference/feature-lifecycle.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Feature Lifecycle
+---
+
+:::info Availability
+
+Feature lifecycle was introduced in Unleash 6.0.0
+
+:::
+
+![Feature Lifecycle example](/img/feature-lifecycle.png 'Feature Lifecycle example')
+
+Feature Lifecycle facilitates tracking the progression of features through various stages in their development and operational phases. 
+This systematic approach helps in identifying potential bottlenecks and improving the overall efficiency of the software development process.
+
+## Lifecycle stages
+
+Each stage in the lifecycle has a defined purpose:
+
+### Initial stage
+
+The feature flag is created but remains unseen in any environment.
+
+**Purpose:** marks the initiation of the feature.
+
+### Pre-live stage
+
+The feature flag is used in non-production environment or is attempted to be used in a disabled production environment.
```

Comment:
The feature flag has received metrics in non-production environment or has received metrics in production environment, which is disabled.